.cargo/config.toml

```diff
@@ -1,4 +1,7 @@
 [build]
 # We compile with `panic=abort`, so we need `-Cforce-unwind-tables=y`
 # to get a useful backtrace on panic.
-rustflags = ["-Ctarget-feature=+sse4.1,+sse4.2", "-Cforce-unwind-tables=y"]
\ No newline at end of file
+rustflags = ["-Cforce-unwind-tables=y"]
+
+[target.'cfg(target_arch = "x86_64")']
+rustflags = ["-Ctarget-feature=+sse4.1,+sse4.2", "-Cforce-unwind-tables=y"]
```

Comment:
Does the `force-unwind-tables` need to be duplicated?

Reply:
yes, I've explicitly verified the emitted `rustc` commands.

also found the confirming issue: https://github.com/rust-lang/cargo/issues/5376